src/renderers/dom/client/eventPlugins/SimpleEventPlugin.js

```diff
@@ -131,22 +140,22 @@ var topLevelEventsToDispatchConfig = {};
 
 var onClickListeners = {};
 
-function getDictionaryKey(inst) {
+function getDictionaryKey(inst: ReactInstance): string {
   // Prevents V8 performance issue:
   // https://github.com/facebook/react/pull/7232
   return '.' + inst._rootNodeID;
 }
 
-var SimpleEventPlugin = {
+var SimpleEventPlugin: PluginModule<MouseEvent> = {
 
   eventTypes: eventTypes,
 
   extractEvents: function(
-    topLevelType,
-    targetInst,
-    nativeEvent,
-    nativeEventTarget
-  ) {
+    topLevelType: TopLevelTypes,
+    targetInst: ReactInstance,
+    nativeEvent: MouseEvent,
+    nativeEventTarget: EventTarget,
+  ): null | ReactSyntheticEvent {
```

Comment:
Thanks! I would have said the same thing too, except @Aweary pointed out the benefits of `null | <Type>` to me in a different PR. Based on [this post by sebmarkbage](https://www.facebook.com/groups/2003630259862046/permalink/2098952973663107/).